test/srt/model/test_llava_low_api.py

```diff
@@ -89,13 +112,21 @@ def test_generate_worker(
     tp_size,
 ):
     model_config = ModelConfig(path=model_path)
-    model = ModelRunner(model_config, 0.8, tp_rank, tp_size, 28888)
+    model = ModelRunner(
+        server_args=ServerArgs(model_path=model_path),
+        model_config=model_config,
+        mem_fraction_static=0.8,
+        tp_rank=tp_rank,
+        tp_size=tp_size,
+        gpu_id=0,
+        nccl_port=28888,
+    )
     # print(model.model)
 
     # Prepare data
     prompt = "A chat between a curious human and an artificial intelligence assistant. The assistant gives helpful, detailed, and polite answers to the human's questions. USER: <image>\nDescribe this picture ASSISTANT:"
-    image_path = "/home/ubuntu/sglang/test/lang/test_image.png"
-    image = load_image(image_path)
+    image_path = "/content/drive/MyDrive/sglang/test/srt/example_image.png"
```

Comment:
@ZackZeng999  This part of the test is not runnable by anyone other than you. Good to find an non-copyrighted image and put into repo for local testing.